### PR TITLE
Abb 102

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record; finally I decided in favour of describing this manuscript as "composite".

### CODE REVIEW

File: ParisBNF/abb/BNFabb102.xml [hunk 1]
```diff
@@ -0,0 +1,863 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BNFabb102" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Story of Mary, malkǝʾ-hymns and other texts</title>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:8]: Better in transliteration I think, "STory of Mary" sounds a bit generic

File: ParisBNF/abb/BNFabb102.xml [hunk 2]
```diff
@@ -0,0 +1,863 @@
...
+                  <decoNote xml:id="d7" type="miniature">
+                      <locus target="#10r"/>
+                      <desc><ref type="authFile" corresp="AT1022MiraclesOfMary">How the Virgin Mary
+                      expelled a snake from the belly of a man</ref>.</desc>
+                  </decoNote>
+
+                  <decoNote xml:id="d8" type="miniature">
+                      <locus target="#12r"/>
+                      <desc><ref type="authFile" corresp="AT1022MiraclesOfMary">How the Virgin Mary
+                      raise a man from the death</ref>.</desc>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:286]: raised

File: ParisBNF/abb/BNFabb102.xml [hunk 3]
```diff
@@ -0,0 +1,863 @@
...
+                  <decoNote xml:id="d10" type="miniature">
+                      <locus target="#14r"/>
+                      <desc><ref type="authFile" corresp="AT1022MiraclesOfMary">How the Virgin Mary healed
+                      sick people who hosted her</ref>.</desc>
+                  </decoNote>
+
+                  <decoNote xml:id="d11" type="miniature">
+                      <locus target="#15r"/>
+                      <desc><ref type="authFile" corresp="AT1022MiraclesOfMary">How the Virgin Mary
+                      healed the an whose flesh was cut</ref>.</desc>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:304]: "an"?

File: ParisBNF/abb/BNFabb102.xml [hunk 4]
```diff
@@ -0,0 +1,863 @@
...
+                        corresp="AT1022MiraclesOfMary">
+                        How the Archangels
+                        punished the sorcerer</ref>.</desc>
+                  </decoNote>
+
+                  <decoNote xml:id="d26" type="miniature">
+                      <locus target="#36r"/>
+                      <desc><ref type="authFile"
+                        corresp="AT1022MiraclesOfMary">
+                        How the childred of the old man returned to him</ref>.</desc>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:408]: children

File: ParisBNF/abb/BNFabb102.xml [hunk 5]
```diff
@@ -0,0 +1,863 @@
...
+                        corresp="AT1022MiraclesOfMary">
+                        How the man called Bārok
+                        (cp. <ref type="work" corresp="LIT3612Miracle"/>) was beaten by robbers and saved by the Virgin Mary</ref>.
+                        </desc>
+                  </decoNote>
+
+                  <decoNote xml:id="d39" type="miniature">
+                      <locus target="#68r"/>
+                      <desc><ref type="authFile"
+                        corresp="AT1022MiraclesOfMary">
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:504]: AT1134MMAnestasyos

File: ParisBNF/abb/BNFabb102.xml [hunk 6]
```diff
@@ -0,0 +1,863 @@
...
+                        How the Virgin Mary appeared to
+                        <persName ref="PRS4587Georget"/>George the New</ref> in the shape of a dove.</desc>
+                  </decoNote>
+
+                  <decoNote xml:id="d38" type="miniature">
+                      <locus target="#64r"/>
+                      <desc><ref type="authFile"
+                        corresp="AT1022MiraclesOfMary">
+                        How the man called Bārok
+                        (cp. <ref type="work" corresp="LIT3612Miracle"/>) was beaten by robbers and saved by the Virgin Mary</ref>.
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:497]: AT1133MMBarok

File: ParisBNF/abb/BNFabb102.xml [hunk 7]
```diff
@@ -0,0 +1,863 @@
...
+            </desc>
+            </handNote>
+            </handDesc>
+                           <decoDesc>
+                             <decoNote xml:id="d61" type="miniature">
+                                 <locus target="#52r"/>
+                                 <desc><ref type="authFile"
+                                   corresp="AT1031VirginAndChild">
+                                   The Virgin and Child, the prone figure of the owner
+                                    depicted beneath</ref>.</desc>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:772]: I would not use note here, but write the entire description inside desc

File: ParisBNF/abb/BNFabb102.xml [hunk 8]
```diff
@@ -0,0 +1,863 @@
...
+
+
+          </physDesc>
+              <history>
+              <origin>
+                <origDate notBefore="1700" notAfter="1820">18th or early 19th century</origDate>.
+                <ref target="#p2">Unit 2</ref> is represented by
+                a part of one sole single leaf mounted on the leaf <locus target="#52"/> (possibly originally left
+                blank) of <ref target="#p1"/>,
+                which bear the miniature <ref target="#d61"/> on one side and the beginning
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb102.xml:787]: bears

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: ParisBNF/abb/BNFabb102.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb102.xml:8]: ok

File: ParisBNF/abb/BNFabb102.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb102.xml:304]: done

File: ParisBNF/abb/BNFabb102.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb102.xml:408]: ok

File: ParisBNF/abb/BNFabb102.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb102.xml:504]: ok

File: ParisBNF/abb/BNFabb102.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb102.xml:497]: ok

File: ParisBNF/abb/BNFabb102.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb102.xml:772]: done

**Review by thea-m — APPROVED** [review]

**Pull request merged**